src/Raven.Server/Commercial/LetsEncryptClient.cs

```diff
@@ -295,14 +315,13 @@ public async Task CompleteChallenges(CancellationToken token = default(Cancellat
                 string responseText;
                 try
                 {
-                    (result, responseText) = await SendAsync<AuthorizationChallengeResponse>(HttpMethod.Post, challenge.Url, new AuthorizeChallenge
-                    {
-                        KeyAuthorization = _jws.GetKeyAuthorization(challenge.Token)
-                    }, token);
+				    // From: https://tools.ietf.org/html/rfc8555#section-7.5.1
+                    (result, responseText) = await SendAsync<AuthorizationChallengeResponse>(HttpMethod.Post, challenge.Url, "{}", token);
 
                     while (result.Status == "pending")
                     {
-                        (result, responseText) = await SendAsync<AuthorizationChallengeResponse>(HttpMethod.Get, challenge.Url, null, token);
+					    // post-as-get (https://community.letsencrypt.org/t/acme-v2-scheduled-deprecation-of-unauthenticated-resource-gets/74380)
+                        (result, responseText) = await SendAsync<AuthorizationChallengeResponse>(HttpMethod.Post, challenge.Url, string.Empty, token);
```

Comment:
few lines up you have `{}` as content, not here. Is this correct?

Reply:
yes that's correct

Reply:
Ok, thanks for explanation.